### DISCUSSION TITLE
Script Kit for Windows - Developer Preview

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#654] [Announcements]
# Script Kit for Windows - Developer Preview

## Why "Developer Preview"

1. I haven't bought a certificate to add to the build:
- You'll see many "untrusted" warnings when downloading/installing
- Auto-updating will not work

2. I haven't decided if the Windows repo will be a fork, branch, or main

Currently, the Windows build builds from the exact same branch as the Mac build. While this works fine, for now, I'm pretty sure we'll want two separate release cadences and feature sets. This could be done through a branch, but then I'd have to set up a new release server. I'll figure it out the details next year.

3. The Windows build is missing all the OS-specific tools

Windows currently doesn't support `getSelectedText`, `getTabs`, and other utils that are written in AppleScript. I'm planning to tackle many of those utils in Rust, which theoretically should allow them to be cross-platform, but that's another task for next year.

4. I've Only Tested It on Two Laptops

The Mac version has been used/tested by many, many people. I have two Windows laptops at home to test it on. It works well, but I don't know how much your mileage will vary.

## Where to Download

Download the  installer here:
https://github.com/johnlindquist/kitapp/releases/tag/v1.5.1

Again, this build will not auto-update. I'll post announcements here when new versions are available and you'll have to download the new version each time until I have the certificate and release servers worked out. Honestly, I'll probably write a "check for Windows update and download" script then you can just run that on a `// Schedule: 0 8 * * *` 😉

**Comment by nadilas** [comment]
This would be great :)
